ls1intum/Artemis
# `Iris`: Add testserver for Iris

### Checklist
#### General



### Motivation and Context


Since we want to keep Iris separate from the develop branch for now, we need a dedicated testserver for Iris. This test server is the only one that should be used to test Iris PRs.

### Description

I've set up TS9 accordingly to work with Iris and added it to the workflow file.
Note: This PR includes the changes of #6572, as TS9 already requires the changes from there. 

### Steps for Testing

1. Deploy to TS9
2. See that the deployment works and Artemis starts
3. If you know what you are doing: Test the endpoints via Postman or smth

### Review Progress






#### Code Review
#### Manual Tests
